### PR TITLE
Animation to About us Section

### PR DESCRIPTION
**Pull request by sohini3062**
In reference to 
Add animation to the about us section #49

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -219,17 +220,18 @@ header {
 
 .about h1 {
     font-size: 50px;
+    
     font-weight: bold;
     backdrop-filter: blur(20px);
 }
 
-hr {
+/*hr {
```
Comment (harikesh409) [inline css/style.css:228]: Is there any particular reason to remove the hr styles? If not uncomment it.

File: css/style.css [hunk 2]
```diff
@@ -219,17 +220,18 @@ header {
 
 .about h1 {
     font-size: 50px;
+
```
Comment (harikesh409) [inline css/style.css:223]: empty line can be removed

File: index.html [hunk 3]
```diff
@@ -95,16 +96,44 @@
...
+                  <ellipse cx="187.01" cy="197.24" rx="6.27" ry="6.71" style="fill: #989fa4"/>
+                </g>
+                <line id="left" x1="118.43" y1="232.25" x2="73.23" y2="224.9" style="fill: #fff;stroke: #0d1932;stroke-miterlimit: 10;stroke-width: 4px"/>
+                <line id="right" x1="257.88" y1="232.25" x2="303.07" y2="224.9" style="fill: #fff;stroke: #0d1932;stroke-miterlimit: 10;stroke-width: 4px"/>
+                <line y1="249.4" x2="374.02" y2="249.4" style="fill: none;stroke: #0e1a32;stroke-miterlimit: 10;stroke-width: 4px"/>
+              </svg>
             <p class="about-content">
-                “Algorithms are not arbiters of objective truth and fairness simply because they're math.” - <i><b>Zoe Quinn</b></i>
-
+
```
Comment (harikesh409) [inline index.html:126]: remove empty line

File: index.html [hunk 4]
```diff
@@ -95,16 +96,44 @@
...
-                “Algorithms are not arbiters of objective truth and fairness simply because they're math.” - <i><b>Zoe Quinn</b></i>
-
+              
+               “Algorithms are not arbiters of objective truth and fairness simply because they're math.-”<i><b>Zoe Quinn</b></i></div>
+          
                 <br> <br>
-                <p class="text-justify">
-                    We are an organization of people who focuses on Data Structure and Algorithms. It is our goal to provide access to well written, well-explained programs and knowledge that improve people's lives. We want to enable people to adapt and change code according
-                    to their own ideas through an Open Access approach. We bring amazing people together to code algorithms and share our work for the benefit of all.
+                <div class="text-justify">
```
Comment (harikesh409) [inline index.html:130]: use the `<p class="text-justify">` only or else correct the spacing if you plan to use the `div`.

**Review by harikesh409 — CHANGES_REQUESTED** [review]

File: index.html (hunk 4 above)
Comment (sohini3062) [reply index.html]: I have added the p class text justify and removed the div

**Comment by sohini3062** [comment]
Will I create another branch with the new files? I am not sure if the changes I made are viewable

**Comment by surajbugade** [comment]
Hello @harikesh409  . Why my pull request #200 was not merged and you created a similar pull request #201 and merged it? I'd also created an issue before opening the PR.

**Comment by harikesh409** [comment]
> Hello @harikesh409 . Why my pull request #200 was not merged and you created a similar pull request #201 and merged it? I'd also created an issue before opening the PR.

I have not seen that PR and it is just a revert so as soon as I opened the site it was messed up and I immediately reverted the #98 pr by creating a new one.

**Comment by surajbugade** [comment]
> > Hello @harikesh409 . Why my pull request #200 was not merged and you created a similar pull request #201 and merged it? I'd also created an issue before opening the PR.
> 
> I have not seen that PR and it is just a revert so as soon as I opened the site it was messed up and I immediately reverted the #98 pr by creating a new one.

Okay! I just wanted to know. Looking forward to contribute something good in the project. Thanks 😊 .

**Comment by harikesh409** [comment]
> > > Hello @harikesh409 . Why my pull request #200 was not merged and you created a similar pull request #201 and merged it? I'd also created an issue before opening the PR.
> > 
> > 
> > I have not seen that PR and it is just a revert so as soon as I opened the site it was messed up and I immediately reverted the #98 pr by creating a new one.
> 
> Okay! I just wanted to know. Looking forward to contribute something good in the project. Thanks 😊 .

All the best. 👍

**Pull request merged**